src/Raven.Server/Documents/ETL/EtlProcess.cs

```diff
@@ -431,7 +403,7 @@ public bool Load(IEnumerable<TTransformed> items, DocumentsOperationContext cont
 
                     stats.RecordLastLoadedEtag(stats.LastTransformedEtags.Values.Max());
 
-                    Statistics.LoadSuccess(stats.NumberOfTransformedItems.Sum(x => x.Value));
+                    Statistics.LoadSuccess(count);
```

Comment:
It's for stats only but I don't think that the usage of `count` is correct. `LoadInternal` returns the number of commands (for example we might have DELETE and PUT for a single transformed item) while we should record `transformed - filtered` items here

Reply:
To me it is not well defined.

does sending a tombstone counts as `Load` ? (it doesn't count as `Transformed`, it counts as `NumberOfTransformedTombstones`)
Load of a time-series is segment based, entries based or is it count as 1 regardless to the number of segments (`TransformedItems` is segment based btw.)
Same for counters.

Not sure that `count` is the best choice though..

Reply:
As we discussed there are other problems with using the transformed number of items - for example we might transform time series segment but filter out all it's entries under the covers.

So the usage of `count` had some drawbacks but appears to be the best number for stats purpose.